src/main/blackbox/blackbox.h

```diff
@@ -43,11 +43,11 @@ typedef enum BlackboxMode {
 } BlackboxMode;
 
 typedef enum FlightLogEvent {
-    FLIGHT_LOG_EVENT_SYNC_BEEP = 0,
+    FLIGHT_LOG_EVENT_SYNC_BEEP           = 0,
```

Comment:
I am strongly against enforcing 'alignment' like this - this will introduce unneeded churn every time the longest element in an aligned list is added or removed.

Reply:
We can turn that off. Looking at the existing code, it looked like there were more enums aligned than unaligned. Note that either we align all enums or either we align none. What would be better?

Reply:
Not force-aligning or force-not-aligning them is the way to go in my opinion.